node/src/accountant/mod.rs

```diff
@@ -1541,7 +1557,7 @@ mod tests {
             actual_prepared_adjustment
                 .original_setup_msg
                 .protected_qualified_payables,
-            protect_payables_in_test(affordable_accounts.clone())
+            initial_unadjusted_accounts
```

Comment:
Previously it was adjusted accounts (inside `affordable_accounts`), now it is  unadjusted payments (inside ` initial_unadjusted_accounts`). I don't understand what exactly changed in the production code.

Reply:
This is a correction of a mistake I did before. We say "affordable accounts" or "payments" about the result of the adjustment, however here we are looking at sort of the input of the process so we should not use the word affordable now, they definitely are not affordable because that's the whole point of the PaymentAdjuster: if we've got right here, it means, we cannot afford to pay these out.